cloudwatch_sns_to_slack/handler.py

```diff
@@ -115,14 +120,18 @@ def _get_channel(sns_topic_arn):
 
     if topic_name[:6] != slack_topic_prefix:
         logger.warning(f'Slack messaging behavior not defined for SNS topic {sns_topic_arn}.')
-        logger.info('Will send to #platform-alerts...')
-        return PLATFORM_ALERTS_CHANNEL
+        logger.info(f'Will send to {SRE_ALERTS_CHANNEL}...')
+        return SRE_ALERTS_CHANNEL
 
     channel_name = '#' + topic_name.split(slack_topic_prefix)[-1]
     logger.info(f'Will send message to channel {channel_name}')
     return channel_name
 
 
+def _upload_to_s3(record):
+    S3_CLIENT.put_object(Body=json.dumps(record), Bucket=S3_BUCKET_NAME, Key=f'{str(uuid4())}.json')
```

Comment:
The Key is the file name, right? Could it be easier to debug if we had the name as something like `{channel}_{utcnow()}.json`? Or maybe just `{utcnow()}.json`

Reply:
yup, it's the filename--mmm i get upload timestamp from the object metadata, so would rather not add it in a different place. i can pull out the channel name, think that'll make it easier to get general impressions just by glancing at the bucket, but then I think i'll just use a UUID to distinguish